### PR TITLE
st_geometry extents bug

### LINKED ISSUE #377: `struct(xmin, ymin, xmax, ymax)` should be accepted in st_geometry function

### PR DESCRIPTION
**Pull request by vpipkt**
Eventually fix #377

### CODE REVIEW

File: pyrasterframes/src/main/python/tests/RasterFunctionsTests.py [hunk 1]
```diff
@@ -96,6 +96,15 @@ def test_general(self):
 
         df.first()
 
+    def test_st_geometry_from_struct(self):
+        from pyspark.sql import Row
+        from pyspark.sql.functions import struct
+        df = self.spark.createDataFrame([Row(xmin=0, ymin=1, xmax=2, ymax=3)])
+        df.select(st_geometry(struct(df.xmin, df.ymin, df.xmax, df.ymax)).alias('geom'))
```
Comment (metasim) [inline pyrasterframes/src/main/python/tests/RasterFunctionsTests.py]: I hate that you have to `struct` an already named `Row`....

Comment (vpipkt) [reply pyrasterframes/src/main/python/tests/RasterFunctionsTests.py]: hm maybe you could just pass the whole row in ? idk.

Comment (metasim) [reply pyrasterframes/src/main/python/tests/RasterFunctionsTests.py]: @vpipkt No, there's a bug in PySpark + Python 3.6 that makes field order non-deterministic.

**Comment by metasim** [comment]
Related: https://stackoverflow.com/questions/35343525/how-do-i-order-fields-of-my-row-objects-in-spark-python/35343885#35343885

**Pull request merged**